install/plugin-updater.php

```diff
@@ -409,6 +409,10 @@ public function plugins_api_filter( $_data, $_action = '', $_args = null ) {
 			$_data = $edd_api_request_transient;
 		}
 
+		if ( empty( $_data ) ) {
+			return $_data;
+		}
+
```

Comment:
As you proposed in the PR description, I think it's better to check `$api_response` and make an early return if it's `false`.

Indeed, in addition of `get_version_from_remote()` could return `false` [here](https://github.com/polylang/polylang/blob/master/install/plugin-updater.php#L628) and [here](https://github.com/polylang/polylang/blob/master/install/plugin-updater.php#L636), `api_request()` could also return false [here](https://github.com/polylang/polylang/blob/master/install/plugin-updater.php#L502) and [here](https://github.com/polylang/polylang/blob/master/install/plugin-updater.php#L506) and `NULL` [here](https://github.com/polylang/polylang/blob/master/install/plugin-updater.php#L497).

Reply:
Yep, see https://github.com/polylang/polylang/pull/1714/commits/22353ea1fe458749a45e66ef67d1e7f4c2dd5416.